examples/aks/main.tf

```diff
@@ -2,10 +2,10 @@ terraform {
   required_providers {
     azurerm = {
       source  = "hashicorp/azurerm"
-      version = "~> 2.57"
+      version = "~> 2.67"
     }
   }
-  required_version = "~> 0.15.0"
+  required_version = "~> 1.0.0"
```

Comment:
Just `~> 1.0` should be safe to use now that TF has reached v1.x, it means any version <2.x will work.

`~> 1.0.0` means any version <1.1.x.